opal/mca/btl/openib/btl_openib.h

```diff
@@ -149,6 +150,12 @@ typedef struct mca_btl_openib_srq_manager_t {
 } mca_btl_openib_srq_manager_t;
 #endif
 
+typedef enum {
+    MCA_BTL_IB_XRC_API_NONE,
+    MCA_BTL_IB_XRC_API_BETA,
+    MCA_BTL_IB_XRC_API_OFED_3_12
+} mca_btl_openib_xrc_api_t;
```

Comment:
Is it possible to use names that are not dependent on specific versions?

E.g., what does "beta" refer to?  Is that a beta of a specific OFED (or MOFED) version?

Is it possible to have names that reflect specific API characteristics?